frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Edit_HPA_Action.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Edit HPA action
+	As a user, I want to edit the HPA assigned to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment 
+    And user has added CPU resource limit
+    And user has added Memory resource limit
+    And user is in topology
+
+@regression
+Scenario: Edit HPA action form view
+   Given user has a workload with HPA assigned to it
+   When user opens sidebar of workload
+   And user opens action menu
+   And user clicks on Edit Horizontal Pod Autoscaler option 
+   And user sees Edit Horizontal Pod Autoscaler form opens
+   And user sees values are prefilled
+   And user checks the name value but can not edit it
+   And user checks and edit the value for maximum and minimum pods
+   And user checks and edit the cpu value
+   And user checks and edit the memory value
+   And user clicks on save to get back to topology page
+   And user opens the sidebar
+   And user sees Horizontal Pod Autoscalers section
+   And user goes to the details tab
```

Comment:
It should be above this "And user sees Horizontal Pod Autoscalers section" statement, isn't it?

Reply:
No, It would be resource tab